ray_on_golem/client/client.py

```diff
@@ -192,3 +203,62 @@ def _make_request(
             raise RayOnGolemClientValidationError(
                 "Couldn't validate response data",
             ) from e
+
+    def _start_webserver(self) -> None:
+        with cli_logger.group("Ray On Golem webserver"):
```

Comment:
the name of the logging group should most likely be a constant defined somewhere...